cognite/client/_api/functions.py

```diff
@@ -721,6 +719,8 @@ def _validate_and_parse_requirements(requirements: List[str]) -> List[str]:
     parsed_reqs: List[str] = []
     for req in requirements:
         try:
+            from pip._internal.req.constructors import install_req_from_line
```

Comment:
Don't import inside the try/except, or the import error gets transformed into a ValueError. You're also reimporting the module  for every iteration in the loop.

We also have a local_import function in utils which will raise an exception with a more friendly error message. I'd prefer if you used that.

The `except Exception` is also bad practice. But that can be addressed in a different PR.